cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -29,8 +29,8 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/emicklei/go-restful"
 	"github.com/golang-jwt/jwt"
-	"github.com/gorilla/mux"
```

Comment:
Is there other part using `github.com/gorilla/mux` in KubeEdge?

Reply:
only edge/test/cloudhub/stub.go
https://github.com/kubeedge/kubeedge/blob/f4420cd3a354d8a8f9ef02fd8d195ca2d0eeab3a/edge/test/cloudhub/stub.go#L8

And github.com/docker/distribution still use it